st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -447,6 +463,19 @@ def main():
             print('Aborting')
             sys.exit(1)
 
+    # Prompt user for optional additional context info
+    user_info = {}
+
+    print('If you want us to get back to you via email, you can provide additional context '
```

Comment:
I'd just say "Please provide us your name, email and an optional comment if you want us to contact you about this report". Wdyt?